scripts/homepage.js

```diff
@@ -230,19 +238,21 @@ $(function () {
         var translate_download = $('#translate-download').text();
         var translate_purchase = $('#translate-purchase').text();
 
-        if (
-            $('input#amount-custom').hasClass('checked') &&
-            payment_amount < payment_minimum &&
-            $('#amount-custom').val() != ''
-        ) {
+        if ($('#amounts').children().length <= 1) {
             $('#download').text(translate_download);
-        } else {
+        } else if (
+            $('button.payment-button').hasClass('checked') ||
+            payment_amount >= payment_minimum
```

Comment:
Change to `$('#amount-custom').val() * 100 >= payment_minimum` and drop the variable, there's no point assigning something we only ever use once, and sometimes not at all.